pkg/virtualKubelet/apiReflection/reflectors/incoming/pods_test.go

```diff
@@ -1,12 +1,18 @@
 package incoming_test
 
 import (
+	"bytes"
+	"errors"
+	"flag"
+	"fmt"
+
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
 	"github.com/onsi/gomega/types"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
```

Comment:
```suggestion
	"k8s.io/klog/v2"
```

Unfortunately we never managed to make this a linter constraint